app/models/vita_partner.rb

```diff
@@ -50,12 +51,9 @@ class VitaPartner < ApplicationRecord
   def at_capacity?
     return parent_organization.at_capacity? if site?
 
-    return false if capacity_limit.blank?
+    return false if capacity_limit.blank? || organization_capacity.active_client_count.nil?
```

Comment:
Since you call `organization_capacity.active_client_count` twice in this method, you might be doing the query twice. Not a blocker, but fyi.